### ISSUE TITLE
Updates from Play store when installed via F-Droid.

### ISSUE THREAD

**Issue by smith901** [Crazy-Marvin/Morse#55]
Like the title says. Play did a security update an hour ago, 1800UTC. No updates pending in F-Droid or Play Store. Wouldn't have caught except I was about to use my phone when I saw the update notification. For whatever reason, Play Store update notifications aren't persistent like the other apps.
What's really going on with this app?

**Comment by CrazyMarvin** [comment]
I am not sure if I understand your point. 🤔
It is a normal behavior that Google Play updates apps installed via F-Droid and vice versa. The last update was some weeks ago though and not recently.

**Comment by smith901** [comment]
> The last update was some weeks ago though and not recently.    

Yeah, your last sentence. F-Droid says last update was version 1.5 on June 3, 2020. Today, I saw the Play Store download notification, which only said _"security_ update". It didn't specify which app so I ran Installalogs, also on F-Droid, which gave this SHA256 hash: `eeb633fe07e501ad2ac4ed8ebcb81366d17531faaf6e92f8309e23feb04fc327`    
Your app wasn't the only one that Play Store did this for today. Ten app total, including Play Store, had a security update installed today without any app having an _"actual"_ update.    
All phones I've had with F-Droid installed couldn't update an app install via Play using F-Droid. I've had to uninstall then reinstall using F-Droid.     
Since you haven't pushed any updates and are confused about my inquiry, I take it that it is Google being Google and doing something.
Time for the full Android FOSS switch for me. I've been seeing these "background updates-not updates" for a few months and was curious so I had to ask when I saw an F-Droid installed app get a Play Store update without any updates pending on either platform.
